### PR TITLE
Blog Post: Matt Morehouse LTS

### PR DESCRIPTION
**Pull request by dergigi**
To be reviewed.

---

Live preview:

- [/blog/matt-morehouse-lightning-security-lts-grant](https://os-website-git-matt-morehouse-lts-open-sats.vercel.app/blog/matt-morehouse-lightning-security-lts-grant) 👀

Social preview:

Screenshot 2023-10-20 at 15 25 54

### CODE REVIEW

**Review by Arvin21M — APPROVED** [review]
Looks great! All external links work. Grammar/punctuations seems all good. Love seeing this particular LTS grant. For novel "tech", security typically takes a back seat to other "sexy" things. Security is a foundational cornerstone (stepping down from soap box now).

File: data/blog/matt-morehouse-lightning-security-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- Various bug fixes and fuzz tests for [CLN](https://github.com/ElementsProject/lightning/commits?author=morehouse) and [LND](https://github.com/lightningnetwork/lnd/commits?author=morehouse).
+
+In all likelihood, the Lightning Network will play a crucial role in any future
+that has Bitcoin scaling to 10 billion users. However, since the beginning of
+the network, the development of new features often had priority over the overall
+security and robustness of the network. Adding new features like dual funded
+channels, Taproot channels, splicing, and BOLT 12 is still the main priority for
+most developers and users, but all these features introduce added complexity.
+"Complexity is the worst enemy of security," to quote Bruce Schneier, "and our
+systems are getting more complex all the time." Including Lightning.  Matt's
```
Comment (morehouse) [inline data/blog/matt-morehouse-lightning-security-lts-grant.mdx:38]: I think this wording flows a little better:

```suggestion
systems are getting more complex all the time." Lightning is no exception.  Matt's
```

**Review by morehouse — APPROVED** [review]
Looks good!

One minor suggestion inline.  Feel free to use it or ignore.

File: data/blog/matt-morehouse-lightning-security-lts-grant.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Long-Term Support For Matt Morehouse"
+date: '2023-10-21'
```
Comment (dergigi) [inline data/blog/matt-morehouse-lightning-security-lts-grant.mdx:3]:
```suggestion
date: '2023-10-23'
```

**Pull request merged**